lib/metadata/index.js

```diff
@@ -262,6 +281,67 @@ Metadata.prototype.getReplicas = function (keyspaceName, tokenBuffer) {
   return keyspace.replicas[closestToken];
 };
 
+/**
+ * Gets the token ranges that define data distribution in the ring.
+ *
+ * @returns {Set<TokenRange>} The ranges of the ring.
+ */
+Metadata.prototype.getTokenRanges = function () {
+  return this.tokenRanges;
+};
+
+/**
+ * Gets the token ranges that are replicated on the given host, for
+ * the given keyspace.
+ *
+ * @param {String} keyspaceName The name of the keyspace to get ranges for.
+ * @param {Host} host The host.
+ * @returns {Set<TokenRange>} Ranges for the keyspace on this host
```

Comment:
We can include that we return `null` when the keyspace is not found in the driver metadata.